requirements/requirements_nlp.txt

```diff
@@ -8,10 +9,10 @@ gdown
 h5py
 ijson
 jieba
-mamba-ssm==2.2.2; sys_platform == 'linux'
+mamba_ssm @ git+https://github.com/state-spaces/mamba.git@v2.2.2
```

Comment:
Set it to sys_platform == linux

Reply:
we actually have to move this into `reinstall.sh` since the Pypi package doesn't build all c-headers when done in CPU-only environments. We learned that it's safer to always build from source and so we think its best to have this in the `reinstall.sh` script.